test-case/run-all-tests.sh

```diff
@@ -228,7 +228,7 @@ test_multiple-pipeline-capture()
 }
 test_multiple-pause-resume()
 {
-	"$mydir"/multiple-pause-resume.sh -r 25
+	"$mydir"/multiple-pause-resume.sh -l "$small_loop" -r 25
```

Comment:
not quite able to understand what you are trying to do?

what does 'faster transition' mean? a reduction in the min/max range? a reduction of both min and max?
And what is this small_loop doing here?

Reply:
I'm trying to add a parameter for loop count.

faster transition by reductions of both min and max. This is nothing but sleep time, bounded by min and max, between pause and resume.